### PR TITLE
Update wallet add funds ts

### PR DESCRIPTION
**Pull request by Anchello**

### CODE REVIEW

File: packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx [hunk 1]
```diff
@@ -75,42 +85,34 @@ class WalletAddFundsForm extends Component {
...
-      depositAddress: PropTypes.string,
-      currency: PropTypes.string,
-      rateToGVT: PropTypes.number,
-      title: PropTypes.string,
-      logo: PropTypes.string
-    })
-  ),
-  t: PropTypes.func
-};
+export default translate()(WalletAddFundsForm);
```
Comment (pavelzubov) [inline packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx]: Change to `const WalletAddFundsForm = ...`

File: packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx [hunk 2]
```diff
@@ -75,42 +85,34 @@ class WalletAddFundsForm extends Component {
...
+export interface CurrentWallet {
+  currency: string;
+  available: number;
+}
+
+interface OwnProps {
+  wallets: WalletData[];
+  currentWallet: CurrentWallet;
+  notifySuccess(x: string): void;
+  notifyError(x: string): void;
```
Comment (pavelzubov) [inline packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx]: Change `x` to correct name

File: packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx [hunk 3]
```diff
@@ -75,42 +85,34 @@ class WalletAddFundsForm extends Component {
...
-      }
-      return {
-        currency,
-        amount: ""
-      };
-    }
-  })
-)(WalletAddFundsForm);
+export interface CurrentWallet {
+  currency: string;
```
Comment (pavelzubov) [inline packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx]: Now for currency props we can use `CurrencyEnum`

File: packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx [hunk 4]
```diff
@@ -75,42 +85,34 @@ class WalletAddFundsForm extends Component {
...
+
+interface OwnProps {
+  wallets: WalletData[];
+  currentWallet: CurrentWallet;
+  notifySuccess(x: string): void;
+  notifyError(x: string): void;
+}
+
+interface State {
+  currency: string;
```
Comment (pavelzubov) [inline packages/shared/modules/wallet-add-funds/components/wallet-add-funds-form.tsx]: Now for currency props we can use `CurrencyEnum`

**Comment by Anchello** [comment]
Fixed according to the comments

**Pull request merged**